src/core/components/link/styles.ts

```diff
@@ -50,27 +51,23 @@ export const subdued = css`
 `
 
 export const icon = css`
-	/*
-	FIXME: This makes icons sit vertically centred, but breaks links with long
-	text descriptions onto a separate line to surrounding text
-	 */
-	display: inline-flex;
 	svg {
 		fill: currentColor;
-		position: absolute;
+		vertical-align: text-top;
```

Comment:
It's admirable to try and avoid hardcoding here, but to my eye the icons are riding a little high. I think I'd prefer to fix the alignment in a hacky way in the short term with a view to fixing it properly when we've spent more time thinking about the rules of icon spacing.

```suggestion
		/*
		TODO: hardcoded bottom margin to vertically align
		icons with text. This needs to be revisited when
		the rules of icon spacing have been formalised
		 */
		margin-bottom: -3px;
```

For a visual comparison:

`vertical-align: text-top`

![Screenshot 2020-10-19 at 12 03 08](https://user-images.githubusercontent.com/5931528/96442515-29c65e00-1203-11eb-9d6b-5732c8365012.png)

`margin-bottom: -3px`

![Screenshot 2020-10-19 at 12 02 14](https://user-images.githubusercontent.com/5931528/96442509-26cb6d80-1203-11eb-9983-7478a13d13bc.png)

Reply:
That looks better, I agree that consistent icon spacing would make this easier!